npm/src/index.ts

```diff
@@ -42,6 +43,7 @@ export const renderModules = async () => {
   const fileNavigation = document.querySelector(".govuk-tna-tree")
   const timeoutDialog: HTMLDialogElement | null =
     document.querySelector(".timeout-dialog")
+  const multiSelectSearch = document.querySelector(".tna-multi-select-search")
```

Comment:
Should update the unit tests to make sure the new component is called correctly: `/npm/test/index.test.ts`